tools/tokamak_grids/gridgen/process_grid.pro

```diff
@@ -311,7 +311,7 @@ FUNCTION my_int_y, var, yaxis, mesh, loop=loop, nosmooth=nosmooth, simple=simple
       ;; Only set loop integral in closed (periodic) domains i.e. the
       ;; core. Otherwise it may be overwritten by values in a PF region
        
-      loop[xi] = f[xi,yi[N_ELEMENTS(yi)-1]] - f[xi,yi[0]]
+      loop[xi] = f[xi,yi[N_ELEMENTS(yi)-1]] - f[xi,yi[0]] - f[xi,yi[0]] + 0.5D*(yaxis[xi,yi[1]]-yaxis[xi,yi[0]])*(var[xi,yi[N_ELEMENTS(yi)-1]] + var[xi,yi[0]])
```

Comment:
Is `- f[ix,yi[0]]` supposed to be in there twice?

Reply:
Sorry, my bad. I'll fix it.